libs/live-signer-evm/src/DmkSignerEth.ts

```diff
@@ -0,0 +1,166 @@
+import { lastValueFrom } from "rxjs";
+import {
+  GetAddressDAError,
+  SignerEth,
+  SignerEthBuilder,
+  SignPersonalMessageDAError,
+  SignTransactionDAError,
+  SignTypedDataDAError,
+} from "@ledgerhq/device-signer-kit-ethereum";
+import {
+  DeviceActionState,
+  DeviceActionStatus,
+  DeviceManagementKit,
+  hexaStringToBuffer,
+} from "@ledgerhq/device-management-kit";
+import { EIP712Message } from "@ledgerhq/types-live";
+import { EthAppPleaseEnableContractData, UserRefusedOnDevice } from "@ledgerhq/errors";
+import { EvmAddress, EvmSignature, EvmSigner } from "@ledgerhq/coin-evm/types/signer";
+import { LoadConfig, ResolutionConfig } from "@ledgerhq/hw-app-eth/lib/services/types";
+
+export type DAError =
+  | GetAddressDAError
+  | SignTypedDataDAError
+  | SignTransactionDAError
+  | SignPersonalMessageDAError;
+
+export class DmkSignerEth implements EvmSigner {
+  private readonly signer: SignerEth;
```

Comment:
[ASK] I don't understand the difference between the new EvmSigner interface and the SignerEth interface

Reply:
`EvmSigner` is the interface used in ledger live, `SignerEth` is the signer interface from the dmk signer. `SignerEth` does not have the same signature as it use device actions. So this class wrap the dmk signer to adapt it for ledger live

Reply:
Ah ok. I didn't notice this PR was on the ledger-live repo 😅